developer/src/tike/project/Keyman.Developer.System.Project.ProjectFile.pas

```diff
@@ -621,8 +621,8 @@ function TProjectFile.IsSourceFile: Boolean;
     Exit(True);
 
   // Only return true if the file is directly in the ProjectOptions.SourcePath folder
-  SourcePath := ReplaceStr(IncludeTrailingPathDelimiter(FProject.ResolveProjectPath(FProject.Options.SourcePath)), '/', '\');
-  FilePath := ReplaceStr(ExtractFilePath(FFileName), '/', '\');
+  SourcePath := DosSlashes(FProject.ResolveProjectPath(FProject.Options.SourcePath));
```

Comment:
Just checking this intentionally no longer has `IncludeTrailingPathDelimiter`?

Reply:
Yes, because `ResolveProjectPath` does that now. (See https://github.com/keymanapp/keyman/pull/10016/files#diff-123b0a7530c20259d4b04d281a9a8dcd1c2051f83b8298f789f516f8d4495984R1231)